resume.tex

```diff
@@ -0,0 +1,123 @@
+
+\input{command.tex}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Now for the actual document:
+
+\begin{document}
+
+\input{header.tex}
+
+%%%%%%%%%%%%%%%%%%%%%%%% 
+\begin{ressection}{Employment}
+	\begin{restitledposition}{Nurx / Thirty Madison}{Staff Software Engineer}{April 2022--August 2024}
+		\ressubitem{\textbf{Job Runner} Designed, implemented, and maintained an in-house asynchronous job running
+		solution. Using RabbitMQ, NestJS, and NodeJS/TypeScript, created a new standardized solution for running async
+		tasks at scale across a suite of microservices.}
+		\ressubitem{\textbf{Kafka Topic Isolation} Isolated Kafka topics within staging environments
+		leading to improved stability across dozens of staging environments leading to increased developer efficiency.}
+		\ressubitem{\textbf{Health Metric Standardization} Combining an internal NPM package, Terminus, and Kubernetes
+		probes developed a reliable set of metrics to monitor service health.}
+		\ressubitem{\textbf{Core Services} Supported a microservices architecture with tools and libraries including:
+		RabbitMQ, Kafka, NestJS, GraphQL, Prisma, Postgres, RoR, and several AWS services.}
+		\ressubitem{\textbf{Architecture Review Group} Led an architecture review group for upcoming major technical projects.
+		Participated in intake/scoping, led review discussion, and fascilitated feedback to engineers.}
```

Comment:
used "led" twice. Leadership is an important quality to emphasize; maybe use an antonym.

Reply:
antonym? did you mean synonym?

Reply:
nope, you should put "followed micromanaged assignments" lol. yes. yes I meant synonym.

Reply:
Ok, wasn't sure if there was some wordsmith play by using an antonym in a double negative or idk what. haha, ok

Reply:
I'm also going to move this up under job runner. The first is an independently driving org wide effort, the second is a leadership role. the other points are good to leave but are secondary.